packages/lib/services/search/SearchEngine.ts

```diff
@@ -602,7 +602,12 @@ export default class SearchEngine {
 	}
 
 	private normalizeText_(text: string) {
-		const normalizedText = text.normalize ? text.normalize() : text;
+		let normalizedText = text.normalize ? text.normalize() : text;
+
+		// Null characters can break FTS. Remove them.
+		// eslint-disable-next-line no-control-regex
+		normalizedText = normalizedText.replace(/\x00/g, ' ');
```

Comment:
Should it be a space, or should it be an empty string? I'm wondering because normally a null character will render to nothing?

Reply:
I chose a space so that it still serves as a word break. (Normally NUL characters render to red dots (in CodeMirror 5) or something similar in CodeMirror 6).